### PR TITLE
Update eg-ie-etc-nb.md

### PR DESCRIPTION
**Pull request by gazanson**
Edits as per Content Guide Google doc  (https://docs.google.com/document/d/1mnkOQCR-FhziM3-bWFRYJogjjzUBVGgzCbZu9YBT4cA/edit?ts=59c1ac45#)

### CODE REVIEW

**Comment by joolswood** [comment]
Looks good. We do try to use real examples if we can. Consider this (probably not the best example, maybe you can come up with something better!):

> Try to restructure the sentence so the user does not need to guess what may be missing. For example, 'We help people to improve their writing skills including grammar, spelling and research.'

**Comment by gazanson** [comment]
@joolswood Think this may be good to go now? Appears to be stopped in the PR queue .... Thanks.

File: _punctuation-grammar/eg-ie-etc-nb.md [hunk 1]
```diff
@@ -4,14 +4,22 @@ order: 7
...
 
-Only do this if your users can understand it (sometimes screen readers won't read it correctly).
+### ie
 
-Avoid using 'that is' or 'meaning' as an alternative to &#8217;ie&#8217;. Rewrite and restructure your content instead.
+Avoid using ‘that is’ or ‘meaning’ as an alternative to ‘ie’. Rewrite and restructure your content instead.
+
+### etc
+
+Try to restructure the sentence so the user does not need to guess what may be missing. For example, 'We help people to improve their writing skills including grammar, spelling and research.
```
Comment (joolswood) [inline _punctuation-grammar/eg-ie-etc-nb.md]: Needs closing ' on example, or better still convert both examples to example boxes?

**Comment by joolswood** [comment]
Closing this until it comes up in sprint.

**Pull request closed without merging**